pkg/secrethub/client.go

```diff
@@ -102,7 +104,7 @@ func NewClient(with ...ClientOption) (*Client, error) {
 
 	// Try to use default key credentials if none provided explicitly
 	if client.decrypter == nil {
-		err := WithCredentials(credentials.UseKey(nil, nil))(client)
+		err := WithCredentials(credentials.UseKey(client.DefaultCredential()))(client)
 		// nolint: staticcheck
 		if err != nil {
 			// TODO: log that default credential was not loaded.
```

Comment:
TODO found

Reply:
Is this what we want to pick up now though? We do not have any logging in place on the client at the moment.

Reply:
I don't know, just spotted the TODO and thought I'd bring it up. Your call what to do here.

Reply:
We decided not to do it right now.

Reply:
#119